docs/getting-started/user-guides/paymetheus.md

```diff
@@ -17,7 +17,8 @@ You're now ready to start using Decred! Start the 'Decred' program. You will now
 ![Paymetheus connection screen](../../img/Paymetheus-dcrd-login.png)  
 
 >Two programs are actually running when you start Decred. There's 'Paymetheus' which is the graphical wallet we'll talk further about
->and 'dcrd' which is the communications program that actually talks to the Decred network. This means that you can run dcrd
+>and 'dcrd' which is the communications daemon (pronounced DAY-mon; a program that runs in the background and that does not interact
```

Comment:
"A program that is not invoked explicitly" -- www.dictionary.com

"In multitasking computer operating systems, a daemon (/ˈdiːmən/ or /ˈdeɪmən/) is a computer program that runs as a background process, rather than being under the direct control of an interactive user." -- Wikipedia

I think it fits the definition of 'daemon'. Again, keep in mind this is novice user documentation. I didn't want to use the word 'daemon' at all since few people outside of the tech sphere would know what it is, but some think it's a good idea to keep it constant across user and technical docs. 

Your example above shows this anyway. you're not interacting with the daemon, you're interacting with a control program that in turn communicates with it, which is the very definition of the word.